presto-docs/src/main/sphinx/presto_cpp/properties.rst

```diff
@@ -144,3 +144,41 @@ The configuration properties of Presto C++ workers are described here, in alphab
 
   Number of drivers to use per task. Defaults to hardware CPUs.
 
+Build-enabled Properties
+----------------
+The following properties are available only if certain properties are enabled at build time via CMakeLists.txt.
```

Comment:
```suggestion
The following properties are available only if certain properties are enabled at build time by using CMakeLists.txt.
```
Do not use via. See the [GitLab documentation style guide recommended word list entry for via](https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#via) for alternatives: I suggested one but please choose another rephrasing if "by using" doesn't work for you.